pkg/ddevapp/ddevapp_test.go

```diff
@@ -235,6 +235,20 @@ var (
 			DynamicURI:                    testcommon.URIWithExpect{URI: "/", Expect: "This is a test page"},
 			FilesImageURI:                 "/sites/default/files/2022-07/Logo.png",
 		},
+		// 13: craftcms
+		{
+			Name:                          "TestPkgCraftCms",
+			SourceURL:                     "https://github.com/khalwat/ddev/files/9497111/cms-4.2.3.zip",
```

Comment:
Please add a comment about how to recreate this zipball for a new version of craft.

Reply:
@bencroker you mind taking care of that?

Reply:
Will do.

Reply:
This is complete.